code/modules/uplink/uplink_item.dm

```diff
@@ -1222,7 +1222,6 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	desc = "A tattered old jumpsuit that will provide absolutely no benefit to you. It fills the wearer with a strange compulsion to blurt out 'glorf'."
 	item = /obj/item/clothing/under/color/grey/glorf
 	cost = 20
-	surplus = 0
```

Comment:
Glorf suit is a useless 20tc item that should always be barred from surplus crate as well